eq/util/texture.cpp

```diff
@@ -304,11 +305,18 @@ void Texture::bindToFBO( const GLenum target, const int32_t width,
     _generate();
 
     EQ_GL_CALL( glBindTexture( _impl->target, _impl->name ));
-    EQ_GL_CALL( glTexImage2D( _impl->target, 0, _impl->internalFormat, width, height, 0,
-                              _impl->format, _impl->type, 0 ));
+    EQ_GL_CALL( glTexImage2D( _impl->target, 0, _impl->internalFormat, width,
+                              height, 0, _impl->format, _impl->type, 0 ));
     EQ_GL_CALL( glFramebufferTexture2DEXT( GL_FRAMEBUFFER, target,
                                            _impl->target, _impl->name, 0 ));
 
+    if( samples > 1 )
```

Comment:
Ah, got it. One sample = no multisampling. eq::ON is 1, is this translated to non-1 somewhere?